rmgpy/tools/loader.py

```diff
@@ -39,19 +39,23 @@
 from rmgpy.solver.liquid import LiquidReactor
 from rmgpy.solver.base import TerminationConversion
 
-def loadRMGJob(inputFile, chemkinFile=None, speciesDict=None, generateImages=True, useJava=False, useChemkinNames=False):
+def loadRMGJob(inputFile, chemkinFile=None, speciesDict=None, generateImages=True, useJava=False,
```

Comment:
Same comment here, should the default option be False? There are others here in this commit as well.

My opinion is that if the function can only be used on RMG generated mechanisms, then by default we should not check for duplicates. If there is even the slightest chance though that a non-RMG mechanism could be used, then I think we should check for duplicates (to avoid risk of error) and update the documentation in this case to tell RMG mechanism users to set this flag to not check for duplicates.

Reply:
See above comment